core/tests/unit/ldml/keyboards/k_103_vkey.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+@@keys: [K_A][K_U][K_Q][K_C][K_K][K_SLASH][K_Z][K_Q][K_L][K_T][K_W]
+@@expected: \u0071\u0075\u0061\u0063\u006b\u002f\u0077\u0061\u006c\u0074\u007a
+-->
+
+<!DOCTYPE keyboard SYSTEM "../../../../../resources/standards-data/ldml-keyboards/techpreview/dtd/ldmlKeyboard.dtd">
+<keyboard locale="en" conformsTo="techpreview">
+  <info author="srl295" indicator="🙀" layout="azerty" normalization="NFC" />
```

Comment:
should `layout="azerty"` imply this vkey mapping?

Reply:
`layout="azerty"` is purely informational.